torch/_dynamo/symbolic_convert.py

```diff
@@ -652,36 +651,6 @@ def call_function(
             raise AssertionError(f"Attempt to trace forbidden callable {inner_fn}")
         self.push(fn.call_function(self, args, kwargs))
 
-    def update_locals_and_stack(self, oldvar: VariableTracker, newvar: VariableTracker):
-        def repl(v: VariableTracker):
-            if v.mutable_local is oldvar.mutable_local:
-                return newvar
-            return v
-
-        recursive_parents = oldvar.parents_tracker.recursive_parents()
-
-        def skip(v: VariableTracker):
-            return v.parents_tracker not in recursive_parents
-
-        cache: Dict[int, Tuple[object, object]] = dict()
-        self.output.side_effects.apply(repl, cache, skip_fn=skip)
```

Comment:
Running repl on side effects here, aside from doing __dict__ updating and transfer, also ensured that source was correct for when we did oldvar->newvar. Maybe this is not possible with a general removal of replace_all, but it would be worth triple checking this.